file.c

```diff
@@ -5227,50 +5227,25 @@ rb_thread_flock(void *data)
  *  call-seq:
  *    flock(locking_constant) -> 0 or false
  *
- *  Locks or unlocks a file according to the given `locking_constant`,
+ *  Locks or unlocks file +self+ according to the given `locking_constant`,
  *  a bitwise OR of the values in the table below.
  *
  *  Not available on all platforms.
  *
  *  Returns `false` if `File::LOCK_NB` is specified and the operation would have blocked;
  *  otherwise returns `0`.
+ *
  *  <br>
```

Comment:
Is there a reason why we should keep the line break tags before and after the table?

Reply:
I had to experiment to get any vertical whitespace at all;  this is what I came up with.